apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -87,8 +91,13 @@ export type BaseNewsFormProps = {
     OnCompletedMsg: (INewsItem) => ArgsProps | null,
     allNews: INewsItem[],
     actionName: ActionNameProps,
+    totalProperties: number
 }
 
+const OptionalBlock = styled.div<{ show?: boolean }>`
+  ${({ show }) => !show ? 'display: none;' : ''}
+`
+
```

Comment:
You don't really need to use styled for that small functionality. Instead of this you can use React conditional rendering syntax

Reply:
So there is no reason for CPU optimization, if you write this feature for that reason. If organization had only one property -> visibility of this block will be calculated once

Reply:
We need to render both elements, but only one should be visible.
This is necessary in order not to rewrite the monstrous existing logic